google-cloud-compute/src/main/java/com/google/cloud/compute/nativeimage/ComputeFeature.java

```diff
@@ -0,0 +1,3657 @@
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetTcpProxyList$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGateway");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGateway$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGateway$Status");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGatewayAggregatedList");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGatewayAggregatedList$Builder");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TargetVpnGatewayAggregatedList$ItemsDefaultEntryHolder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGatewayList");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGatewayList$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGatewaysScopedList");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TargetVpnGatewaysScopedList$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestFailure");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestFailure$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestIamPermissionsDiskRequest");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestIamPermissionsDiskRequest$Builder");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsExternalVpnGatewayRequest");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsExternalVpnGatewayRequest$Builder");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsFirewallPolicyRequest");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsFirewallPolicyRequest$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestIamPermissionsImageRequest");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestIamPermissionsImageRequest$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestIamPermissionsInstanceRequest");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsInstanceRequest$Builder");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsInstanceTemplateRequest");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsInstanceTemplateRequest$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestIamPermissionsLicenseCodeRequest");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsLicenseCodeRequest$Builder");
+    computeClassesBuilder.add("com.google.cloud.compute.v1.TestIamPermissionsLicenseRequest");
+    computeClassesBuilder.add(
+        "com.google.cloud.compute.v1.TestIamPermissionsLicenseRequest$Builder");
+    computeC
```

Comment:
Out of curiosity, what happens when we remove these lines?

Reply:
You get the errors such as:

```
       Suppressed: java.lang.ExceptionInInitializerError
         com.google.cloud.compute.v1.Compute.<clinit>(Compute.java:18809)
         com.google.cloud.compute.v1.Operation.getDescriptor(Operation.java:291)
         com.google.cloud.compute.v1.stub.HttpJsonRegionInstanceGroupManagersStub.<clinit>(HttpJsonRegionInstanceGroupManagersStub.java:84)
         com.google.cloud.compute.v1.RegionInstanceGroupManagersClientTest.startStaticServer(RegionInstanceGroupManagersClientTest.java:58)
         java.lang.reflect.Method.invoke(Method.java:566)
         [...]
       Caused by: java.lang.RuntimeException: Generated message class "com.google.api.FieldBehavior" missing method "valueOf".
         com.google.protobuf.GeneratedMessage.getMethodOrDie(GeneratedMessage.java:1973)
         com.google.protobuf.GeneratedMessage.access$1100(GeneratedMessage.java:61)
         com.google.protobuf.GeneratedMessage$GeneratedExtension.<init>(GeneratedMessage.java:1789)
         com.google.protobuf.GeneratedMessage.newFileScopedGeneratedExtension(GeneratedMessage.java:1643)
         com.google.api.FieldBehaviorProto.<clinit>(FieldBehaviorProto.java:55)
         [...]
       Caused by: java.lang.NoSuchMethodException: com.google.api.FieldBehavior.valueOf(com.google.protobuf.Descriptors$EnumValueDescriptor)
         java.lang.Class.getMethod(DynamicHub.java:1114)
         com.google.protobuf.GeneratedMessage.getMethodOrDie(GeneratedMessage.java:1970)
         [...]
```

https://github.com/googleapis/java-compute/issues/685